### PR TITLE
460: Add tooltips to the accept/reject/fuzzy buttons.

### LINKED ISSUE #460: tooltips for "reject" and "add translation" for editor
as editor, i do not know , what they do. i was afraid. i wanted to know, is rejected translation deleted? is it shown in translation history of string? also, same questions in case i edit suggested translation and submit it, what happens with original translation. i asked in chat, ( https://wordpress.slack.com/messages/polyglots/ , https://wordpress.slack.com/messages/glotpress/ ), and known out that they are not deleted and available through filter interface. then i have tried to filter so, and looked history of one of them, and see, they are also shown in history. but i would like to know/see that easily and immediately, through tooltip, or link of help.

### PR DESCRIPTION
**Pull request by toolstack**
Resolves #460.

### CODE REVIEW

**Comment by ocean90** [comment]
Title attributes should be avoided, see https://make.wordpress.org/accessibility/handbook/best-practices/quick-start-guide/#links.

This should make use of the `aria-label` attribute, something like GitHub is doing.

tooltips

**Comment by toolstack** [comment]
GP uses the title in other areas and using `aria-label` requires JavaScript so I'd be against that, at least without the redesign in place.

Since these are buttons and not links, the suggestions to avoid the title attribute doesn't seem to apply.

File: gp-templates/translation-row.php [hunk 1]
```diff
@@ -149,17 +149,17 @@
 					<?php if ( $t->translation_status ): ?>
 						<?php if ( $can_approve_translation ) : ?>
 							<?php if ( 'current' !== $t->translation_status ) : ?>
-							<button class="approve" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $t->id ) ); ?>"><strong>+</strong> <?php _e( 'Approve', 'glotpress' ); ?></button>
+							<button class="approve" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $t->id ) ); ?>" title="<?php echo esc_attr( __( 'Approve this translation.  Any existing translation will be kept as part of the translation history.', 'glotpress' ) ); ?>"><strong>+</strong> <?php _e( 'Approve', 'glotpress' ); ?></button>
```
Comment (ocean90) [inline gp-templates/translation-row.php]: Can you replace `echo esc_attr( __( )` with `esc_attr_e()`? And the double whitespace after the fullstop should be removed.

**Review by ocean90 — CHANGES_REQUESTED** [review]

File: gp-templates/translation-row.php (hunk 1 above)
Comment (toolstack) [reply gp-templates/translation-row.php]: Done.

**Pull request merged**